tests/integration/features/bootstrap/FeatureContext.php

```diff
@@ -555,11 +560,16 @@ public function userSeesAttendeesInRoom(string $user, string $identifier, int $s
 					$attendee['actorId'] .= '@' . rtrim($this->baseRemoteUrl, '/');
 				}
 
+				// Breakout room regex
+				if (isset($attendee['actorId']) && strpos($attendee['actorId'], '/') === 0 && preg_match($attendee['actorId'], $actual['actorId'])) {
+					$attendee['actorId'] = $actual['actorId'];
```

Comment:
Could be good to break a bit this row to make more easy the use of xdebug and prevent wrap the row in editor.
```suggestion
				if (isset($attendee['actorId'])) {
					if (strpos($attendee['actorId'], '/') === 0) {
						if (preg_match($attendee['actorId'], $actual['actorId'])) {
					$attendee['actorId'] = $actual['actorId'];
```

Reply:
With xdebug just inspect the array?
If I split it like this PHPStorm complains that they should be combined into one:
![image](https://user-images.githubusercontent.com/213943/205987316-a6a81657-5d93-457e-8cc2-7522934e91ae.png)

Reply:
PHPStorm isn't a good boy 😂
The problem with xdebug is because is necessary to check condition by condition to identify if the sentence will be true. But, the most important is to work.